test/data_bundle_test.dart

```diff
@@ -27,4 +29,33 @@ void main() async {
       expect(await dataItem.verify(), isFalse);
     });
   });
+
+  test('check if avro serializes tags correctly', () {
```

Comment:
Whitespace between tests. Also consider testing the round trip -> raw string -> serialize -> deserialize -> check against raw string.

Reply:
Done. 
[Also created tech debt ticket ](https://ardrive.atlassian.net/jira/software/c/projects/PE/boards/2?modal=detail&selectedIssue=PE-595)